professional/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('', views.home, name='home'),
+    path('register/', views.register, name='register'),
```

Comment:
You don't need to add the urls.py file for the current demo

Reply:
Done